task: rename cart to order

connoratrug opened the pull request:
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

fdlk commented:
When I push cart save button I get `Invalid [string] value [null] for attribute [orderNumber] of entity [null] with type [lifelines_order].`

fdlk commented:
When I submit order and push edit button in orders page, I get `Unknown entity with 'orderNumber' 'undefined' of type 'order'.`

fdlk commented:
So I think it still needs to account for the id attribute name and auto change

ConnorStroomberg commented:
@fdlk yes thats is the new save func i was working on , this PR is just te change to the cart entity , but i can put in all in one big PR if you like, but in the mean time of stuff will not work as the backend model is updated

fdlk commented:
@connoratrug I think 4040a23 sort of kicks it back into shape for now, shall we merge it like this?

connoratrug commented:
@fdlk i updated the rest of the code to work with orderNumber and implemented the new save everything strategy we discussed with Morris;  https://github.com/molgenis/molgenis-app-lifelines-webshop/pull/108 . Suggest we merge https://github.com/molgenis/molgenis-app-lifelines-webshop/pull/108 and close this one.

connoratrug commented:
closed in favor of : #108
